pilot/pkg/model/extensions.go

```diff
@@ -108,22 +113,36 @@ func convertToWasmPluginWrapper(plugin *config.Config) *WasmPluginWrapper {
 				VmConfig: &envoy_extensions_wasm_v3.VmConfig{
 					Runtime: defaultRuntime,
 					Code:    datasource,
+					EnvironmentVariables: &envoy_extensions_wasm_v3.EnvironmentVariables{
+						KeyValues: map[string]string{
```

Comment:
Yeah it is not obfuscated, although since agent will remove this before forwarding, so it won't be in config dump and we just need to be careful that the resource is not logged by us at istiod and istio-agent.